### PR TITLE
[Core22] hooks: include ppa:ubuntu-security/fde-ice as part of the builds

### PR DESCRIPTION
**Pull request by mvo5**
The security team maintains a PPA with the user-space parts of cryptsetup support for the Inline Crypto Engine (ICE) code.

This needs to be included in the core build so that FDE on ICE is supported.

The full ICE implementation uses:
https://github.com/chrisccoulson/linux/tree/dm-blk-crypto https://github.com/chrisccoulson/cryptsetup/tree/ice-wip

and is getting upstreamed by the kernel and security teams.

### CODE REVIEW

File: hook-tests/001-extra-packages.disabled [hunk 1]
```diff
@@ -134,3 +134,6 @@ if [ -n "$DIFF" ]; then
     echo "test_pkg_removal.sh test."
     exit 1
 fi
+
```
Comment (valentindavid) [inline hook-tests/001-extra-packages.disabled:137]: I think we should re-enable this test...

Comment (mvo5) [reply hook-tests/001-extra-packages.disabled:137]: Excellent point,  started in https://github.com/snapcore/core-base/pull/102

Comment (mvo5) [reply hook-tests/001-extra-packages.disabled:137]: Thanks, I rebased on your PR

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM

**Pull request merged**